src/utils/logger.utils.ts

```diff
@@ -0,0 +1,48 @@
+import winston from 'winston';
+
+//Winston aliases for readability
+const combine = winston.format.combine;
+const timestamp = winston.format.timestamp;
+const prettyPrint = winston.format.prettyPrint;
+const colorize =  winston.format.colorize;
+const align = winston.format.align;
+const printf = winston.format.printf;
+
+const alignedWithColorsAndTime = winston.format.combine(
+  colorize(),
+  timestamp(),
+  align(),
+  printf((info) => {
+    const {
+      timestamp, level, message, ...args
+    } = info;
+
+    const ts = timestamp.slice(0, 19).replace('T', ' ');
+    return `[${ts} ${level}]: ${message} ${Object.keys(args).length ? JSON.stringify(args, null, 2) : ''}`;
+  }),
+);
+
+const logger = winston.createLogger({
+    level: "debug",
+    transports: [
+      new winston.transports.File({
+        filename: "logs/example.log",
+        format: combine(
+          timestamp({
+            format: "MMM-DD-YYYY HH:mm:ss",
+          }),
+          prettyPrint()
+        )
+      }),
+      new winston.transports.Console({
+        format:  combine(
+          timestamp({
+            format: "MMM-DD-YYYY HH:mm:ss",
+          }), 
+          alignedWithColorsAndTime
```

Comment:
maybe im confused - but what's the actual different between the output for prettyPrint and the output for alignedWithColorsAndTime

Reply:
I'll add some comments here in the code
but these are two different transports
winston.transports.File makes a log that saves to file (JSON) and the file needs to be formatted with prettyprint

winston.transports.Console prints to console which needs string formatting with alighedWithConolorsAndTime